faculty_cli/cli.py

```diff
@@ -480,54 +514,65 @@ def server():
     help="Print extra information about servers.",
 )
 def list_servers(project, all, verbose):
-    """List your Faculty servers."""
-    project_id = _resolve_project(project)
+    """List your Faculty servers.
 
+
+    If you do not specify a project, all servers will be listed."""
     status_filter = None if all else ServerStatus.RUNNING
-    servers = _get_servers(project_id, status=status_filter)
+    if not project:
+        projects = {project.id: project.name for project in _list_projects()}
+        user_id = _get_authenticated_user_id()
+        servers = [
+            (server.project_id, projects[server.project_id], server)
+            for server in _list_user_servers(user_id, status=status_filter)
+        ]
+    else:
+        project_id = _resolve_project(project)
+        servers = [
+            (project_id, "", server)
+            for server in _get_servers(project_id, status=status_filter)
+        ]
 
-    if verbose:
-        if not servers:
-            click.echo("No servers.")
-        else:
-            headers = (
-                "Server Name",
-                "Type",
-                "Machine Type",
-                "CPUs",
-                "RAM",
-                "Status",
-                "ID",
-                "Started",
+    status_filter = None if all else ServerStatus.RUNNING
+    headers = (
+        "Project Name",
```

Comment:
I think it make sense to retain the behaviour that the first column is the same as what gets printed when the CLI is used in non-verbose mode. Could we reorder to make "Server Name" first?

Also, it's a little jarring that under the new implementation, the existing use case (when specifying the project) will result in an empty column. In this case, the user will also have two columns (project ID and name) that are all the same and are pushing more useful information to the right (and possibly wrapping in their terminal). Have you considered putting project name and ID at the end and only showing them when no project was specified on the CLI? (Note: I think it would be important to have them at the end in this case so that the ordering of all the other columns is consistent between the all-projects and specified-project cases.